src/main/scala/beam/sim/config/MatSimBeamConfigBuilder.scala

```diff
@@ -113,6 +113,7 @@ class MatSimBeamConfigBuilder(beamConf: Config) extends LazyLogging {
           case None =>
             logger.warn(s"MATSim module '$moduleName' not found")
         }
+      case(moduleSplit, value) => logger.warn(s"Unexpected module split $moduleSplit in MatsimBeamConfigBuilder. Value is $value")
```

Comment:
I'm unsure if this is possible as I did not dig into the config splitpath code, but it seemed possible, so maybe the warning is fine and we can adjust if needed?